### DISCUSSION TITLE
Case insensitive unique constraints

### DISCUSSION THREAD

**Discussion by igormorgado** [collerek/ormar#431] [Q&A] [answered]
I could not find in documentation if is possible to define case insensitive unique constraints. For example if my usernames are email bounded,  EMAIL@domain.com and email@domain.com are the same. 

The other solution is to lowercase everything before put on database.

What are the best recommendations to solve this type of problem?

**Comment by collerek** [comment, marked as answer]
I think right now it's not possible.
This can be an enhancement issue :)
